test/integration/default/controls/config_spec.rb

```diff
@@ -15,6 +15,7 @@
     its('content') { should include '"pkg": {"name": "' }
     its('content') { should include '"service": {"name": "' }
     its('content') { should include '"tofs": {"files_switch": ["any/path/can/be/used/here", "id", "osfinger", "os", "os_family"], "source_files": {"template-config-file-file-managed": ["example.tmpl.jinja"]}' }
+    its('content') { should include '"arch": "amd64"}' }
```

Comment:
This works:

```suggestion
    its('content') { should include '"arch": "amd64"' }
```

Reply:
Thats fixed now. yes go ahead and make commits!! thanks @myii